<?xml version="1.0"?>
<sdf version="1.7">
    <world name="world1">
        <light type="directional" name="sun">
            <cast_shadows>true</cast_shadows>
            <pose>0 0 10 0 0 0</pose>
            <diffuse>0.8 0.8 0.8 1</diffuse>
            <specular>0.2 0.2 0.2 1</specular>
            <attenuation>
                <range>1000</range>
                <constant>0.9</constant>
                <linear>0.01</linear>
                <quadratic>0.001</quadratic>
            </attenuation>
            <direction>-0.5 0.1 -0.9</direction>
        </light>

        <model name="ground_plane">
            <static>true</static>
            <link name="link">
                <collision name="collision">
                    <geometry>
                        <plane>
                            <normal>0 0 1</normal>
                        </plane>
                    </geometry>
                </collision>
                <visual name="visual">
                    <geometry>
                        <plane>
                            <normal>0 0 1</normal>
                            <size>100 100</size>
                        </plane>
                    </geometry>
                    <material>
                        <ambient>0.8 0.8 0.8 1</ambient>
                        <diffuse>0.8 0.8 0.8 1</diffuse>
                        <specular>0.8 0.8 0.8 1</specular>
                    </material>
                </visual>
            </link>
        </model>
        <model name="obstacle1">
        <static>true</static>
        <pose>4.5 1.0 0.5 0 0 0</pose>
        <link name="link">
            <collision name="collision">
            <geometry>
                <box>
                <size>5.0 2.0 1</size>
                </box>
            </geometry>
            </collision>
            <visual name="visual">
            <geometry>
                <box>
                <size>5.0 2.0 1</size>
                </box>
            </geometry>
            <material>
                <ambient>0.1 0.1 0.1 1</ambient>
                <diffuse>0.2 0.01 0 1</diffuse>
                <specular>0.2 0.01 0 1</specular>
            </material>
            </visual>
        </link>
        </model>
        <model name="obstacle2">
        <static>true</static>
        <pose>2.25 6.0 0.5 0 0 0</pose>
        <link name="link">
            <collision name="collision">
            <geometry>
                <box>
                <size>2.5 3.0 1</size>
                </box>
            </geometry>
            </collision>
            <visual name="visual">
            <geometry>
                <box>
                <size>2.5 3.0 1</size>
                </box>
            </geometry>
            <material>
                <ambient>0.1 0.1 0.1 1</ambient>
                <diffuse>0.2 0.01 0 1</diffuse>
                <specular>0.2 0.01 0 1</specular>
            </material>
            </visual>
        </link>
        </model>
        <model name="obstacle3">
        <static>true</static>
        <pose>10.0 6.6 0.5 0 0 0</pose>
        <link name="link">
            <collision name="collision">
            <geometry>
                <box>
                <size>6.0 1.2000000000000002 1</size>
                </box>
            </geometry>
            </collision>
            <visual name="visual">
            <geometry>
                <box>
                <size>6.0 1.2000000000000002 1</size>
                </box>
            </geometry>
            <material>
                <ambient>0.1 0.1 0.1 1</ambient>
                <diffuse>0.2 0.01 0 1</diffuse>
                <specular>0.2 0.01 0 1</specular>
            </material>
            </visual>
        </link>
        </model>
        <model name="bottom_wall">
        <static>true</static>
        <pose>75.0 -0.25 0.25 0 0 0</pose>
        <link name="link">
            <collision name="collision">
            <geometry>
                <box>
                <size>151.0 0.5 0.5</size>
                </box>
            </geometry>
            </collision>
            <visual name="visual">
            <geometry>
                <box>
                <size>151.0 0.5 0.5</size>
                </box>
            </geometry>
            <material>
                <ambient>0.8 0.8 0.8 1</ambient>
                <diffuse>0.8 0.8 0.8 1</diffuse>
                <specular>0 0.02 0.1 1</specular>
            </material>
            </visual>
        </link>
        </model>
        <model name="top_wall">
        <static>true</static>
        <pose>75.0 75.25 0.25 0 0 0</pose>
        <link name="link">
            <collision name="collision">
            <geometry>
                <box>
                <size>151.0 0.5 0.5</size>
                </box>
            </geometry>
            </collision>
            <visual name="visual">
            <geometry>
                <box>
                <size>151.0 0.5 0.5</size>
                </box>
            </geometry>
            <material>
                <ambient>0.8 0.8 0.8 1</ambient>
                <diffuse>0.8 0.8 0.8 1</diffuse>
                <specular>0 0.02 0.1 1</specular>
            </material>
            </visual>
        </link>
        </model>
        <model name="left_wall">
        <static>true</static>
        <pose>-0.25 37.5 0.25 0 0 0</pose>
        <link name="link">
            <collision name="collision">
            <geometry>
                <box>
                <size>0.5 76.0 0.5</size>
                </box>
            </geometry>
            </collision>
            <visual name="visual">
            <geometry>
                <box>
                <size>0.5 76.0 0.5</size>
                </box>
            </geometry>
            <material>
                <ambient>0.8 0.8 0.8 1</ambient>
                <diffuse>0.8 0.8 0.8 1</diffuse>
                <specular>0 0.02 0.1 1</specular>
            </material>
            </visual>
        </link>
        </model>
        <model name="right_wall">
        <static>true</static>
        <pose>150.25 37.5 0.25 0 0 0</pose>
        <link name="link">
            <collision name="collision">
            <geometry>
                <box>
                <size>0.5 76.0 0.5</size>
                </box>
            </geometry>
            </collision>
            <visual name="visual">
            <geometry>
                <box>
                <size>0.5 76.0 0.5</size>
                </box>
            </geometry>
            <material>
                <ambient>0.8 0.8 0.8 1</ambient>
                <diffuse>0.8 0.8 0.8 1</diffuse>
                <specular>0 0.02 0.1 1</specular>
            </material>
            </visual>
        </link>
        </model>
    </world>
</sdf>